src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -10,66 +8,109 @@
 
         assertNull(actual);
     }
 
     @Test
-    void emptyListTest() {
-        List<UserPost> actual = Collections.emptyList();
+    void sort_EmptyList_EmptyList() {
+        List<UserPost> inputList = Collections.emptyList();
         List<UserPost> expected = Collections.emptyList();
 
-        actual = new AuthorPostSorter().sort(actual);
+        List<UserPost> actual = new AuthorPostSorter().sort(inputList, SortOrder.ASC);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void sort_1ItemList_1ItemList() {
+        List<UserPost> inputList = Arrays.asList(userPost1);
+        List<UserPost> expected = Arrays.asList(userPost1);
+
+        List<UserPost> actual = new AuthorPostSorter().sort(inputList, SortOrder.ASC);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void sort_unsorted3Items_sorted3Items() {
+        List<UserPost> inputList = Arrays.asList(userPost1, userPost2, userPost3);
+        List<UserPost> expected = Arrays.asList(userPost3, userPost1, userPost2);
+
+        List<UserPost> actual = new AuthorPostSorter().sort(inputList, SortOrder.ASC);
 
         assertEquals(expected, actual);
     }
 
     @Test
-    void sortCapital() {
-        UserPost userPost1 = new UserPost("Joe Bloggs",
+    void sort_unsorted3Lower_sorted3Lower() {
+        UserPost userPost1 = new UserPost("joe bloggs",
                 OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
                 "Hello World!", 2);
 
-        UserPost userPost2 = new UserPost("Joe Bloggs",
+        UserPost userPost2 = new UserPost("joe bloggs",
                 OffsetDateTime.of(2020, 1, 3, 8, 53, 34, 0, ZoneOffset.UTC),
                 "Another example post.", 1);
 
-        UserPost userPost3 = new UserPost("Jane Smith",
+        UserPost userPost3 = new UserPost("jane smith",
                 OffsetDateTime.of(2020, 3, 12, 13, 22, 12, 0, ZoneOffset.UTC),
                 "An example of a post \nwith lines breaks.", 3);
 
-        List<UserPost> actual = Arrays.asList(userPost1, userPost2, userPost3);
+        List<UserPost> inputList = Arrays.asList(userPost1, userPost2, userPost3);
         List<UserPost> expected = Arrays.asList(userPost3, userPost1, userPost2);
 
-        actual = new AuthorPostSorter().sort(actual);
+        List<UserPost> actual = new AuthorPostSorter().sort(inputList, SortOrder.ASC);
 
         assertEquals(expected, actual);
     }
 
     @Test
-    void sortLower() {
-        UserPost userPost1 = new UserPost("joe bloggs",
+    void sort_unsorted3ItemsDESC_sorted3ItemsDESC() {
+        List<UserPost> inputList = Arrays.asList(userPost1, userPost2, userPost3);
+        List<UserPost> expected = Arrays.asList(userPost1, userPost2, userPost3);
+
+        List<UserPost> actual = new AuthorPostSorter().sort(inputList, SortOrder.DESC);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void sort_unsorted2Num_sorted2Num() {
```

Comment:
In general you want least complicated to most complicated tests as you go down the page, sorting two items is less complicated than sorting three so should probably be higher up